### ISSUE TITLE
Automatic update of submodules

### ISSUE THREAD

**Issue by liskin** [koreader/contrib#22]
Some of the submodules here are slightly out of date. I was wondering if we perhaps want a GitHub Actions workflow that keeps the submodules pointing to the latest?

Something like https://github.com/xmonad/xmonad-web/blob/gh-pages/.github/workflows/submodule.yaml, or use an action from the marketplace, or something.

**Comment by Frenzie** [comment]
I'm not sure if that's necessarily desirable. It could be under development and therefore unstable, or worse. The situation as it is right now means I can confidently say that none of these do anything suspicious because I trust that whoever merged it personally gave it a quick look.

I suppose an action that opened a PR once a month or so would be alright as far as the suspicious behavior goes.

**Comment by liskin** [comment]
That's a good point. I'll try to think of something. (Sooner or later 🙂)

**Comment by Frenzie** [comment]
Right, it could still be in an unstable state though, currently presumed as pinned to a stable state here. I presume that people may not diligently use tags for little plugins. But perhaps I'm overthinking it.

**Comment by OGKevin** [comment]
Automatic MR's can be created with Renovate. Renovate can also be configure to look for git tags and update to a newer tag accordingly. 

https://docs.renovatebot.com/modules/manager/git-submodules/
